### PR TITLE
LPS-178517 Wrong date format in Export file

### PR DESCRIPTION
**Pull request by matijapetanjek**

### CODE REVIEW

**Comment by matijapetanjek** [comment]
ci:forward

**Comment by matijapetanjek** [comment]
ci:stop

**Comment by matijapetanjek** [comment]
ci:test:relevant

**Comment by yanan** [comment]
No failures related to the pr, it can be forward from qa perspective.

File: modules/apps/batch-engine/batch-engine-service/src/test/java/com/liferay/batch/engine/internal/writer/BaseBatchEngineExportTaskItemWriterImplTestCase.java [hunk 1]
```diff
@@ -251,7 +250,7 @@ protected List<Item> getItems() {
 	protected static final List<String> columnFieldNames = Arrays.asList(
 		"createDate", "description", "id", "name_en", "name_hr");
 	protected static final DateFormat dateFormat = new SimpleDateFormat(
-		"yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
```
[the remark this answers is not in the document]
Comment (matijapetanjek) [reply modules/apps/batch-engine/batch-engine-service/src/test/java/com/liferay/batch/engine/internal/writer/BaseBatchEngineExportTaskItemWriterImplTestCase.java:254]: AFAIS dates in REST apis don't have milliseconds info and by default are always in UTC format, is that wrong? I have updated PR not to use the Z constant but let Java resolve the timezone (which should end up with Z).

**Comment by matijapetanjek** [comment]
ci:test:sf

**Comment by matijapetanjek** [comment]
ci:forward

**Comment by matijapetanjek** [comment]
ci:forward

**Comment by matijapetanjek** [comment]
ci:forward:force

**Pull request closed without merging**